### ISSUE TITLE
Worldmaps: support switching music without scripts

### ISSUE THREAD

**Issue by maxteufel** [SuperTux/supertux#514]
Add a way to switch music in worldmaps without using scripts. This will be necessary for the "merged" Icy Island and Forest Island worldmap in a later release.

**Comment by Karkus476** [comment]
@maxteufel What's wrong with scripting?

**Comment by maxteufel** [comment]
@Karkus476 Scripting requires additional work to track state during restarts. Also without proper implementation it might result in issues similar to the sprite change one.

**Comment by Karkus476** [comment]
@maxteufel Fair enough, but we do need to get to the point where everything is scriptable, so that level designers have complete control over the level. A secure way to preserve data through restart might be worth creating?

**Comment by maxteufel** [comment]
@Karkus476 I believe that is a different issue (open a new one, I guess?) and will require careful planning and discussion.

**Comment by Rusty-Box** [comment]
An idea could be using an object similar to the **special tile** where you can select a new music that plays once players touch this object on a worldmap. The already existing **special tile** could be used for that as well, I think.
